.release-please-manifest.json

```diff
@@ -1,6 +1,6 @@
 {
   ".": "0.13.0",
-  "plugin": "0.32.0",
```

Comment:
Hey! Could you revert this change? i.e. set to 0.32.0.

I've got a [release-please](https://github.com/googleapis/release-please) workflow that will automatically bump the version based on the `fix` keyword I've just added to the pul request title 👍